src/Spryker/Client/OauthPermission/OauthPermission/OauthPermissionReader.php

```diff
@@ -85,10 +85,20 @@ protected function extractPermissionsFromOauthToken(OauthAccessTokenDataTransfer
     /**
      * @param string $authorizationToken
```

Comment:
This is also not quite correct now :)
```suggestion
     * @param string|null $authorizationToken
```